pipelines.adoc

```diff
@@ -73,13 +67,14 @@ Alternatively, if you are using an unmanaged cluster, you can set up a persisten
 . Clone the link:https://github.com/kabanero-io/kabanero-pipelines[kabanero-pipeplines repo]
 
   git clone https://github.com/kabanero-io/kabanero-pipelines
+  
+===== Static Persistent Volumes
 
-. Clone the `pv.yaml` file in this repo and apply it.
-
+If you are not running your openshift cluster on public cloud, you can setup the persistent volume statically. For example on the usage of static persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[here]
```

Comment:
**change**
If you are not running your openshift cluster on public cloud, you can setup the persistent volume statically. For example on the usage of static persistent volume provisioning you can go https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[here]

**to**
If you are not running your OpenShift cluster on a public cloud, you can set up a static persistent volume. For an example of how to use static persistent volume provisioning, see  https://github.com/kabanero-io/kabanero-pipelines/blob/master/pipelines/docs/VolumeProvisioning.md#static-persistent-volumes[Static Persistent Volumes].

Reply:
looks good